### PR TITLE
Added DPA capability.  Added forced time bounds.  Added 2 day limit on t...

### PR DESCRIPTION
**Pull request by DanielJMaher**
...ime bounds

### CODE REVIEW

**Comment by birdage** [comment]
@DanielJMaher when it passes ill merge

**Comment by birdage** [comment]
@FBRTMaka take a tool make sure no conflicts with your stuff

**Comment by FBRTMaka** [comment]
Looks good to me!

File: ooiservices/app/uframe/controller.py [hunk 1]
```diff
@@ -275,11 +292,20 @@ def get_uframe_stream_contents_bounded(mooring, platform, instrument, stream_typ
...
 @auth.login_required
-@api.route('/get_csv/<string:stream>/<string:ref>',methods=['GET'])
-def get_csv(stream,ref):
+@api.route('/get_csv/<string:stream>/<string:ref>/<string:start_time>/<string:end_time>/<string:dpa_flag>',methods=['GET'])
+def get_csv(stream,ref,start_time,end_time,dpa_flag):
     mooring, platform, instrument = ref.split('-', 2)
     stream_type, stream = stream.split('_', 1)
-    data = get_uframe_stream_contents(mooring, platform, instrument, stream_type, stream)
+
+    uframe_data_request_limit = current_app.config['UFRAME_DATA_REQUEST_LIMIT']/1440
```
Comment (birdage) [inline ooiservices/app/uframe/controller.py]: @DanielJMaher Can i get a comment in here about what this is

**Pull request merged**